src/app/parcels/ParcelsPage.tsx

```diff
@@ -188,6 +189,16 @@ const ParcelsPage: React.FC<{}> = () => {
         }
     }, [parcelId]);
 
+    useEffect(() => {
+        if (parcelId && tableData.length > 0) {
+            const selectedRow = tableData.find((rowData) => rowData.parcelId === parcelId);
```

Comment:
As discussed, i think we'd want to fetch from supabase to the parcel data, and then the client id from there.

Reply:
Added